react/next/Styled.tsx

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components'
```

Comment:
Using 'styled-components' isn't against the flexibility we want in our components that we are using CSS Modules.

Reply:
Done